src/c++/perf_analyzer/command_line_parser.h

```diff
@@ -100,6 +100,7 @@ struct PerfAnalyzerParameters {
   bool dynamic_concurrency_mode = false;
   bool url_specified = false;
   std::string url{"localhost:8000"};
+  std::string endpoint{""};
```

Comment:
Usability question: should we combine endpoint and service kind so it's a little simpler for the user?

e.g. instead of user needing to specify
```
--service-kind=opeani
--endpoint=v1/chat/completions
```
which is probably hard, they'd have to google some api reference to find that path, maybe instead we have some predetermined service kinds:
```
--service-kind=openai_chat_completions
```

Or perhaps list out all supported endpoints (if that makes sense) in the `--help` or somewhere?

Reply:
What we have is the most flexible (and was already discussed and decided via Neelay before I implemented)
I was planning to note in documentation that only v1/chat/completions was officially supported. In theory they should all work, but aren't tested and confirmed.

Reply:
At the least could we include a link to OpenAI's top level API reference page in our help message so people aren't completely clueless if they wanna send openAI data?

Reply:
We can discuss that during documentation